ImagePickerSheetController/ImagePickerSheetController/ImagePickerSheetController.swift

```diff
@@ -65,6 +65,9 @@ public class ImagePickerSheetController: UIViewController, UITableViewDataSource
         }
     }
     
+    // Limit of selected images, if limit = 1 (previous will be unselected), if limit > 1 = you can't select new images.
```

Comment:
Use triple slash (///) so that Xcode actually shows this in the hints while typing.